### PR TITLE
Removes unused latejoin spawners

### PR DESCRIPTION
**Pull request by GuiltyNeko**
## What Does This PR Do

Latejoin spawners removed. It's code that was unused and I can't imagine anyone deciding to readd it at any point. Something something new mapper bait I dunno. Either way, I can only see this breaking things if someone somehow had a character with their spawn preference set to something other than the arrivals shuttle, but that seems unlikely. Yes I could technically go ahead and just make all latejoins automatically spawn at arrivals instead of checking through prefs, but... that's a bit more effort and outside what I'm trying to achieve.
Blah blah blah I tested it it doesn't break everything, shouldn't runtime because VScode didn't screech at me about anything.

## Why It's Good For The Game

Dunno. Is it?

## Changelog
If players notice these changes, something has gone horribly wrong.

### CODE REVIEW

File: code/game/objects/effects/landmarks.dm [hunk 1]
```diff
@@ -97,28 +97,6 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/awaystart) //Without this away mission
 	spawner_list = GLOB.latejoin
 	return ..()
 
-/obj/effect/landmark/spawner/late/cryo
-	name = "Late Join Cryo"
-
-/obj/effect/landmark/spawner/late/cryo/Initialize(mapload)
-	spawner_list = GLOB.latejoin_cryo
```
Comment (S34NW) [inline code/game/objects/effects/landmarks.dm:104]: Make sure you also remove these lists if appropriate, and any associated code for spawning.

Comment (GuiltyNeko) [reply code/game/objects/effects/landmarks.dm:104]: The "associated code for spawning" is set up to choose whatever is picked in your preferences, which, the default is the arrivals shuttle, and there's no method whatsoever to change that so far as I'm aware. The code for spawning is, from what I can tell, VERY generous in what it allows, to the point where this won't have any effect so long as nobody has these set in their preferences, which they shouldn't anyways, and is an issue for other reasons anyways.

Comment (S34NW) [reply code/game/objects/effects/landmarks.dm:104]: If all additional spawns are being removed, the code should just put you in the shuttle. No need for code that checks your prefs

Comment (GuiltyNeko) [reply code/game/objects/effects/landmarks.dm:104]: That's a bit more work but I'll see if I can do it without too much stuff falling apart.

**Comment by GuiltyNeko** [comment]
Okay I removed a lot of shit and it works I tested it I was able to latejoin without issues and VSC didn't scream runtimes at me. Let me know if there's more code around here I can rip out.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**